### ISSUE TITLE
Cannot delegate task to a windows host anymore

### ISSUE THREAD

**Issue by JLE-ATS** [ansible/awx-ee#128]
### ISSUE TYPE

- Bug Report

### SUMMARY
Since the 11th of this month, I can't delegate a task to a Windows host from AWX.
I have two AWX instances in two separate datacenter. The problem occurs on both of them.
I also have an Ansible server who can still run tasks delegated to a Windows host. So Windows configuration does not seem to be the issue.

The windows host uses the following parameters: 

- ansible_connection: winrm
- ansible_port: 5985
- ansible_winrm_transport: ntlm
- become_method: runas
- user and password are set to an active user with the proper rights.

Here I the things I did to fix the problem :

- Disable firewall on Windows host in case it was a connectivity issue.
- When firewall was active, allowing "edge traversal" on the rule in case the firewall didn't like traffic from OKD/AWX.
- Declare the host in inventory using its FQDN (default was shortname) in case it was a name resolution problem.
- Declare the host in inventory using its IP address (default was shortname) in case it was a name resolution problem.
- Updating and rebooting the Windows host  in case it was a name Windows weirdness.

At this point, I assume this is a bug introduced by a recent update.

### ENVIRONMENT
First env : 

- AWX version: 19.5.0
- AWX deployment target: OKD 4.8
- This instance could run tasks on a Windows host before.

Second env : 

- AWX version: 21.4.0
- AWX deployment target:  OKD 4.11
- This instance was installed when the problem was detected.

### STEPS TO REPRODUCE
Run any job delegated to a Windows host.
In my case, i use the module "community.windows.win_dns_record" to add a DNS record in Active Directory.

### EXPECTED RESULTS
The task should run without any problems.

### ACTUAL RESULTS
This error : 
`ntlm: HTTPConnectionPool(host='XXXX ', port=5985): Max retries exceeded with url: /wsman (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa7da5c36a0>: Failed to establish a new connection: [Errno 113] No route to host'))",`

Where XXXX is the targeted host and not the host the task is delegated to.

**Comment by akus062381** [comment]
hi @JLE-ATS, this appears to be less of a bug report and more of a question. The message "no route to host" indicates that this could be more of a configuration or environment problem. Could you please ask this on our mailing list? See https://github.com/ansible/awx/#get-involved for information for ways to connect with us.

**Comment by JLE-ATS** [comment]
hi @akus062381, thank you for your input.
I have opened a new conversation on the mailing list as you suggested.

Nonetheless, I still struggle to think it is not a bug of some kind. I have used AWX for months without problems.
Also, my AWX instances still can run tasks (direct tasks i mean) on the same windows host. I don't think the problem come from a network issue, despite the error message.

Are you able to reproduce the error ?

Here is the AWX log for the job (Blue is the targeted host and green is the windows host) :
![image](https://user-images.githubusercontent.com/111417188/185305611-e92c74f2-b5e9-4a75-b443-89d72cb016c8.png)
